exercises/gigasecond/canonical-data.json

```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "gigasecond",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
Should this be `2.0.0` according to [our versioning rules](https://github.com/exercism/problem-specifications#major-version-changes)?

Reply:
Yup.  I think so.  Thanks for catching that.